CIP-0000.md

```diff
@@ -0,0 +1,198 @@
+
+## CIP Process
+When we talk about the community from the perspective of the CIP Process, we are talking about all the validators, stakeholders, core-developers, dapp developers, volunteers and active participants involved in the CIP Process.
+
+Rough Consensus, as mentioned in the previous section, is used to measure community sentiment about a particular proposal. Rough Consensus is measured mostly on the All-Core Devs calls, but it can also happen outside of those calls on Github or Celo Forum comments. Rough Consensus will be determined by the Hardfork Coordinator and All-Core Dev Call organizer based on feedback of the network participants during the call. 
+
+
+Following is the process that a successful non-Core CIP will move along:
+```
+[ WIP ] -> [ DRAFT ] -> [ LAST CALL ] -> [ FINAL ]
+```
+
+Following is the process that a successful Core CIP will move along:
+```
+[ IDEA ] -> [ DRAFT ] -> [ LAST CALL ] -> [ ACCEPTED ] -> [ FINAL ]
+```
+Each status change is requested by the CIP author and reviewed by the CIP editors who are specified in the section below. Use a pull request to update the status. Please include a link to where people should continue discussing your CIP such as a Github Issue or a post on the Celo Forum. The CIP editors will process these requests as per the conditions below.
+* **Idea** — Once the champion has asked the Celo community whether an idea has any chance of support, they will write a draft CIP as a  [pull request](https://github.com/celo-org/celo-proposals/pulls). Consider including an implementation if this will aid people in studying the CIP.
+	* ➡️ Draft — If agreeable, CIP editor will assign the CIP a number (generally the issue or PR number related to the CIP) and merge your pull request. The CIP editor will not unreasonably deny a CIP.
+	* ❌ Draft — Reasons for denying Draft status include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing backwards compatibility, or not in keeping with the  Celo values and code of conduct.
+* **Draft** — Once the first draft has been merged, you may submit follow-up pull requests with further changes to your draft until such point as you believe the CIP to be mature and ready to proceed to the next status. A CIP in draft status must be implemented to be considered for promotion to the next status (ignore this requirement for core CIPs).
+	* ➡️ Last Call — If agreeable, the CIP editor will assign Last Call status and set a review end date (review-period-end), normally 14 days later.
+	* ❌ Last Call — A request for Last Call status will be denied if material changes are still expected to be made to the draft. We hope that CIPs only enter Last Call once, so as to avoid unnecessary noise on the RSS feed.
```

Comment:
We don't otherwise mention an RSS feed. Is this something we are going to implement? I might advocate for leaning on the Forum instead, as it has a pretty good system for managing notifications.

Reply:
The RSS feed is found here: https://github.com/ethereum/EIPs

The code base there can adapted to the CIP process so we can have a cool looking website like cips.celo.org with an RSS feed added